src/clj/harja/palvelin/palvelut/budjettisuunnittelu.clj

```diff
@@ -732,7 +743,13 @@
       ;; Käsittele uudet insertoitavat jhk:t
       (when-not (empty? uudet-jhkt)
         (doseq [{:keys [vuosi kuukausi osa-kuukaudesta tunnit tuntipalkka]} uudet-jhkt]
-          (let [uusi-rivi (insert! db
+          (let [tunnit (if (pvm/sama-tai-jalkeen? urakan-alkupvm (pvm/luo-pvm-dec-kk 2022 10 1))
+                         1 ;; Kaikissa -22 tai myöhemmin alkaneissa urakoissa käytetään kokonaishintaa. Yksittäistä tuntia ei enää tallenneta
```

Comment:
kayta-toimenkuvissa-vuosihintaa -> true/false apurifunktio tai muu vastaava (voitaisiin tehdä cljc -puolelle esim. domain.urakat, niin käytettävissä fe & be), jolla voisi tsekata nämä 2022 urakan tiedoista